### PR TITLE
fix(dashboard): Apply UI fixes to email and in-app editor

### PR DESCRIPTION
**Pull request by SokratisVidros**
### What changed? Why was the change needed?

See preview deployment

### CODE REVIEW

File: apps/dashboard/src/components/workflow-editor/in-app-preview.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ export const InAppPreview = (props: InAppPreviewProps) => {
   return (
     <div
       className={cn(
-        'border-foreground-200 to-background/90 pointer-events-none relative mx-auto flex h-full w-full flex-col gap-4 rounded-xl px-2 py-3 shadow-sm',
+        'border-foreground-200 to-background/90 pointer-events-none relative mx-auto flex h-full w-full flex-col gap-4 rounded-xl px-4 py-3 shadow-sm',
```
Comment (SokratisVidros) [inline apps/dashboard/src/components/workflow-editor/in-app-preview.tsx:32]: Fix alignment issues between the body and the preview header. Padding and margin should always be on the layout level, not on each component of the preview.

File: apps/dashboard/src/components/workflow-editor/steps/email/email-tabs.tsx [hunk 2]
```diff
@@ -55,10 +56,16 @@ export const EmailTabs = (props: StepEditorProps) => {
       </header>
       <Separator />
       <TabsContent value="editor" className={tabsContentClassName}>
-        <EmailEditor uiSchema={uiSchema} />
-        <EmailTabsEditSection>
-          <CustomStepControls dataSchema={dataSchema} origin={workflow.origin} />
-        </EmailTabsEditSection>
+        {workflow.origin === WorkflowOriginEnum.NOVU_CLOUD && (
```
Comment (SokratisVidros) [inline apps/dashboard/src/components/workflow-editor/steps/email/email-tabs.tsx:59]: Show sections based on workflow type avoiding redundant empty divs or the dangling Email Acme header

**Review by LetItRock — APPROVED** [review]

File: apps/dashboard/src/components/workflow-editor/steps/email/email-editor.tsx [hunk 3]
```diff
@@ -14,13 +13,10 @@ export const EmailEditor = (props: EmailEditorProps) => {
 
   return (
     <>
-      <EmailPreviewHeader className="p-3" />
-      <EmailTabsEditSection>{subject && getComponentByType({ component: subject.component })}</EmailTabsEditSection>
-
+      <EmailPreviewHeader />
```
Comment (desiprisg) [inline apps/dashboard/src/components/workflow-editor/steps/email/email-editor.tsx:16]: I think removing the `p-3` here made the header appear in different places for the tabs.

Comment (desiprisg) [reply apps/dashboard/src/components/workflow-editor/steps/email/email-editor.tsx:16]: I'm fixing that now.

**Pull request merged**